src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -83,7 +84,7 @@ namespace {lc.Namespace}
                 }
 
                 LoggerClass parent = lc.ParentClass;
-                var parentClasses = new List<string>();
+                List<string> parentClasses = new List<string>();
```

Comment:
>List<string>  [](http://example.com/codeflow?start=16&length=13)

Sorry, it is ok to use `var` if you are doing `new`. The whole point is the code reader want to know the type. With `new` it is already clear. but for other cases it is not and that is why we avoid using `var` there.